### PR TITLE
Update map_data.json

### PR DESCRIPTION
**Pull request by leanke**
a json update to fix map sizes and incorrect map ids in seafoam island

### CODE REVIEW

File: assets/map_data.json [hunk 1]
```diff
@@ -134,7 +134,7 @@
         },
         {
             "id": "10",
-            "name": "Saffron City",
+            "name": "Saffrdon City",
```
Comment (PWhiddy) [inline assets/map_data.json:137]: I'm guessing this was a typo?

**Comment by PWhiddy** [comment]
thanks! did you test this locally? other than the saffron name LGTM

File: assets/map_data.json (hunk 1 above)
Comment (xinpw8) [reply assets/map_data.json:137]: It's a long-standing typo that has never officially been fixed (albeit it is fixed in several of my repos). Search #pokemon-gen-1 chat for saffrdon to see me talking about it.

Comment (PWhiddy) [reply assets/map_data.json:137]: I think it was fixed in this PR?

https://github.com/PWhiddy/pokerl-map-viz/pull/9/files

this new PR seems to be adding it back? if you add a commit fixing it i can merge.

**Comment by PWhiddy** [comment]
thanks!

**Pull request merged**